test/e2e/manifest/manifest.go

```diff
@@ -45,33 +39,10 @@ func PodFromManifest(filename string) (*v1.Pod, error) {
 	return &pod, nil
 }
 
-// RcFromManifest reads a .json/yaml file and returns the rc in it.
-func RcFromManifest(fileName string) (*v1.ReplicationController, error) {
```

Comment:
unused?

Reply:
yes . it is unused .

Reply:
Since the commit d8a70254106b6f2db9a6f381ff9b3a7d89d761f4 `RcFromManifest()` has been unused.